frontend/src/components/QuestionsList.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Progression, Question } from '../api/models'
+import { Divider } from '@equinor/eds-core-react'
+import { Box } from '@material-ui/core'
+import QuestionAndAnswerFormWithApi from './QuestionAndAnswer/QuestionAndAnswerFormWithApi'
+import QuestionActionsListWithApi from './Action/QuestionActionsListWithApi'
+import AnswerSummaryButton from './AnswerSummaryButton'
+import { useParticipant } from '../globals/contexts'
+
+type Props = {
+    questions: Question[]
+    viewProgression: Progression
+    disable: boolean
+    displayActions?: boolean
+    useFacilitatorAnswer?: boolean
```

Comment:
In this context I think it is more accurate to say

```suggestion
    useAnyAnswer?: boolean
```

Reply:
Yes, that was my first naming, but after you explained I felt this name gave more meaning because it explains the reasoning behind the logic (i.e. we are not using users answers on this step, but the answers/summaries from the facilitators).

Reply:
Ok, but then maybe `useOnlyFacilitatorAnswers´?